doc/man5/x509v3_config.pod

```diff
@@ -122,242 +158,249 @@ following PKIX, NS and MS values are meaningful:
-the given value both the cRLissuer and reasons fields are omitted in this case.
+=item relativename
 
-In the single option case the section indicated contains values for each
-field. In this section:
+The value is taken as a distinguished name fragment that is set as the
+value of the nameRelativeToCRLIssuer field.
 
-If the name is "fullname" the value field should contain the full name
-of the distribution point in the same format as subject alternative name.
+=item CRLIssuer
 
-If the name is "relativename" then the value field should contain a section
-name whose contents represent a DN fragment to be placed in this field.
+The value must in the same format as the subject alternative name.
 
-The name "CRLIssuer" if present should contain a value for this field in
-subject alternative name format.
+=item reasons
 
-If the name is "reasons" the value field should consist of a comma
-separated field containing the reasons. Valid reasons are: "keyCompromise",
-"CACompromise", "affiliationChanged", "superseded", "cessationOfOperation",
-"certificateHold", "privilegeWithdrawn" and "AACompromise".
+A multi-value field that contains the reasons for revocation. The recognized
+values are: C<keyCompromise>, C<CACompromise>, C<affiliationChanged>,
+C<superseded>, C<cessationOfOperation>, C<certificateHold>,
+C<privilegeWithdrawn>, and C<AACompromise>.
 
+=back
+
+Only one of B<fullname> or B<relativename> should be specified.
 
 Simple examples:
 
- crlDistributionPoints=URI:http://myhost.com/myca.crl
- crlDistributionPoints=URI:http://my.com/my.crl,URI:http://oth.com/my.crl
+ crlDistributionPoints = URI:http://myhost.com/myca.crl
+
+ crlDistributionPoints = URI:http://my.com/my.crl, URI:http://oth.com/my.crl
 
 Full distribution point example:
 
- crlDistributionPoints=crldp1_section
+ [extensions]
+ crlDistributionPoints = crldp1_section
 
  [crldp1_section]
-
- fullname=URI:http://myhost.com/myca.crl
- CRLissuer=dirName:issuer_sect
- reasons=keyCompromise, CACompromise
+ fullname = URI:http://myhost.com/myca.crl
+ CRLissuer = dirName:issuer_sect
+ reasons = keyCompromise, CACompromise
 
  [issuer_sect]
- C=UK
- O=Organisation
- CN=Some Name
+ C = UK
+ O = Organisation
+ CN = Some Name
 
 =head2 Issuing Distribution Point
 
-This extension should only appear in CRLs. It is a multi valued extension
+This extension should only appear in CRLs. It is a multi-valued extension
 whose syntax is similar to the "section" pointed to by the CRL distribution
-points extension with a few differences.
-
-The names "reasons" and "CRLissuer" are not recognized.
-
-The name "onlysomereasons" is accepted which sets this field. The value is
-in the same format as the CRL distribution point "reasons" field.
-
-The names "onlyuser", "onlyCA", "onlyAA" and "indirectCRL" are also accepted
-the values should be a boolean value (TRUE or FALSE) to indicate the value of
-the corresponding field.
+points extension, with the following differences. First,
```

Comment:
Is there a better wording here?

...to the "section" pointer to by the CRL distribution _points_ extension, ...

Reply:
I rewrote this to use explicit "=item" lists.